andela/ah-fulldeck
# #162948977 Write unit tests for ALL existing functionality

#### What does this PR do?

Have tests for the existing functionality

#### Description of Task to be completed?

Have tests for the Login functionality 
Have tests for the Signup functionality 
Have tests for the User model 

#### How should this be manually tested?
After cloning the repo using `git clone https://github.com/andela/ah-fulldeck.git`
Move into the project folder `cd <path_of_the_project>`
Check out this branch `ch-test-existing-functionality-162948977` using the command
`git checkout ch-test-existing-functionality-162948977`
Run the tests using the command `python manage.py test`

#### What are the relevant pivotal tracker stories?

[#162948977 ](https://www.pivotaltracker.com/story/show/162948977)

#### Screenshots 
<img width="930" alt="screenshot 2019-01-15 at 15 18 18" src="https://user-images.githubusercontent.com/43740193/51180218-dbbb5280-18d8-11e9-8be3-06b540a1a3de.png">
